controllers/types.go

```diff
@@ -27,3 +27,8 @@ type TokenRequest struct {
 	Avatar       string `json:"avatar"`
 	RefreshToken string `json:"refresh_token"`
 }
+
+const (
+	ReCaptchaVerifySite = "https://recaptcha.net/recaptcha/api/siteverify"
```

Comment:
A captcha implmentation should not be provided in controllers or object package. You should follow the idp package design